tests/AMSlib/ams_allocate.cpp

```diff
@@ -58,9 +55,9 @@ int main(int argc, char* argv[])
   auto& ams_rm = ams::ResourceManager::getInstance();
   ams_rm.init();
   if (device == 1) {
-    if (test_allocation(AMSResourceType::DEVICE, "DEVICE") != 0) return 1;
+    if (test_allocation(AMSResourceType::AMS_DEVICE, "DEVICE") != 0) return 1;
```

Comment:
is 1 an error? use semantic names, e.g.,
```
#define RET_SUCCESS 0
#define RET_ERROR 1
```
(or constexpr if you want to avoid the preprocessor)

Reply:
Fixed.